package.json

```diff
@@ -33,7 +33,12 @@
         "storybook": "dev-pm start storybook",
         "test": "pnpm --recursive run test",
         "version": "$npm_execpath changeset version && pnpm install --lockfile-only",
-        "publish": "pnpm run build:packages && $npm_execpath changeset publish"
+        "publish": "pnpm run build:packages && $npm_execpath changeset publish",
+        "browser": "run-p browser:api browser:admin browser:site browser:jaeger",
```

Comment:
Shall we prefix this with `demo` - I thought it is not needed.

Reply:
We do have a prefix for the `dev`-scripts, so maybe it would be a good idea here as well?

https://github.com/vivid-planet/comet/pull/3036/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R21-R25

We could also add `browser:docs` and `browser:storybook`. However, docs and Storybook automatically open in the browser, so maybe it isn't necessary?

Reply:
Alternatively, the browser scripts could be in demo/package.json. This would be consistent to the Starter.

Reply:
prefixed it with `demo:browser`